client/my-sites/sites/index.jsx

```diff
@@ -35,7 +35,7 @@ export class Sites extends Component {
 
 		// No support for Gutenberg on VIP or Jetpack sites yet.
```

Comment:
>or Jetpack sites yet.

The code comment is now out of sync with code  🙀

Reply:
Well spotted! #32760 👍